javav2/usecases/creating_amazon_cognito_app/Readme.md

```diff
@@ -0,0 +1,327 @@
+# Using Amazon Cognito to require a user to log into a web application
+
+Amazon Cognito lets you add user sign-up, sign-in, and access control to your web applications quickly and easily. Amazon Cognito scales to millions of users and supports sign-in with social identity providers, such as Facebook, Google, and Amazon, and enterprise identity providers such as OAuth2. In this AWS tutorial, OAuth2 and Amazon Cognito are used to protect a web application. This means a user has to log into the application by using credentials of a user defined in an Amazon Cognito User Pool. For example, when a user accesses a web application, they see a web page that lets anonymous users view a log in page, as shown in this illustration.   
```

Comment:
Delete "quickly and easily".
"using credentials" -> "using the credentials"
"this illustration" -> "the following illustration"

Reply:
fixed